src/components/ebay-list/component-browser.ts

```diff
@@ -0,0 +1,17 @@
+import type { WithNormalizedProps } from "../../global";
+export interface ListItem extends Omit<Marko.Input<"item">, `on${string}`> {
+    leading?: Marko.AttrTag<Marko.Input<"div">>;
+    trailing?: Marko.AttrTag<Marko.Input<"div">>;
+}
+
+interface ListInput extends Omit<Marko.Input<"div">, `on${string}`> {
+    "on-item-click"?: (event: {index: number}) => void;
```

Comment:
Sorry for the nitpick, but please put this at the end. We have event handlers at the end as a convention for all other components.